content/docs/operating/security.md

```diff
@@ -177,6 +181,24 @@ and bandwidth.
 It is recommended to monitor all components for failure, and to have them
 automatically restart on failure.
 
+## Server-side TLS and authentication
```

Comment:
This doesn't mention about basic-auth, and we should probably put in a warning about using that over the clear at least.

Reply:
Added basic auth